backend/src/main/java/gov/cdc/usds/simplereport/api/apiuser/UserAccountCreationController.java

```diff
@@ -107,22 +108,20 @@ public void enrollVoiceCallMfa(
       throws OktaAuthenticationFailureException {
     String userId = getUserId(request.getSession());
     String factorId = _oktaAuth.enrollVoiceCallMfa(userId, requestBody.getUserInput());
-    request.getSession().setAttribute(FACTOR_ID, factorId);
+    request.getSession().setAttribute(FACTOR_ID_KEY, factorId);
```

Comment:
I don't think factors have been verified to persist in the spring-session table, but user ids have been verified, and I don't think there'd be any differences? Since they're being set the same way.